src/machine.rs

```diff
@@ -32,7 +32,7 @@ impl<'m> Machine<'m> {
     /// Create a new machine.
     ///
     /// The machine is not started yet.
-    #[instrument]
+    #[instrument(skip_all)]
     pub async fn create(mut config: Config<'m>) -> Result<Machine<'m>, Error> {
```

Comment:
This is a good change but I think we should include the `config` in a trace event so it gets to the log once for both `create` and `connect`.

Reply:
not sure, since caller has control over config, and can log it if necessary